Documentation/docs.md

```diff
@@ -38,7 +37,20 @@ Administrators who need to create reliable and scalable key-value stores for the
  - [Disaster recovery][recovery]
  - [Performance][performance]
  - [Versioning][versioning]
- - [Supported platform][supported_platform]
+
+### Platform guides
+
+ - [Supported systems][supported_platforms]
+ - [docker container][container_docker]
+ - [rkt container][container_rkt]
+ - [Amazon Web Services][aws_platform]
+ - [FreeBSD][freebsd_platform]
+
+### Upgrading and compatibility
+
+ - [Migrate applications from using API v2 to API v3][v2_migration]
+ - [Updating v2.3 to v3.0][v3_upgrade]
```

Comment:
maybe "Updating v2.3 clusters to v3.0" to distinguish from the application migration process? (soft suggest)